editor/script_create_dialog.cpp

```diff
@@ -588,7 +594,7 @@ void ScriptCreateDialog::_bind_methods() {
 	ClassDB::bind_method("_path_entered", &ScriptCreateDialog::_path_entered);
 	ClassDB::bind_method("_template_changed", &ScriptCreateDialog::_template_changed);
 
-	ClassDB::bind_method(D_METHOD("config", "inherits", "path"), &ScriptCreateDialog::config);
+	ClassDB::bind_method(D_METHOD("config", "inherits", "path", "built_in_enabled"), &ScriptCreateDialog::config);
```

Comment:
You should add `DEFVAL(true)` at the end too.

Reply:
Added that.